docs/source/install-django-project.rst

```diff
@@ -16,61 +16,100 @@ participation. For instance, you could contribute an improvement to how
 studies are displayed to participants or create a new CSV format for
 downloading data as a researcher.
 
-   Note: These instructions are for Mac OS. Installing on another OS?
+.. note::
+   These instructions are for Mac OS. Installing on another OS?
    Please consider documenting the exact steps you take and submitting a
    PR to the lookit-api repo to update the documentation!
 
-Basic installation
-~~~~~~~~~~~~~~~~~~~~
-
-Note: the ``$`` represents the command prompt below - e.g. the start of a new line in Terminal. Don't actually type it in!
-
-- Clone the lookit-api repo: ``$ git clone https://github.com/lookit/lookit-api.git``
-- Navigate to the root project directory: ``$ cd lookit-api``
-- Use ``$ pipenv --version`` to see if you already have pipenv installed. If it is, you 
-  will see the version; if not, you will see "pipenv: command not found". If needed, 
-  install using ``pip install pipenv``.
-- Create a virtual environment using pipenv and Python 3.8: ``$ pipenv --python 3.8``
-  If you don't have Python 3.8 available, you can download it from 
-  https://www.python.org/downloads/.
-- Enter the virtual environment using ``$ pipenv shell``.
-- Install invoke using ``$ pip install invoke``.
-- Use the invoke script to go through setup: ``$ invoke setup`` This will install dependencies,
-  create a local .env file, create local SSL certificates, and set up a postgresql database.
-  You will be prompted a few times to enter your password, which is because a command is 
-  being run using ``sudo`` - this should be the password you use
-  to log in to your account on your computer. When it finishes, you should see something like:
+Requirements
+~~~~~~~~~~~~
+
+This is the software you will need to have installed to run lookit-api locally.
+
+
+#. If you haven't already, `install Brew <https://brew.sh>`__ and install Graphviz:
+   
+   .. code-block:: shell
+
+      brew install graphviz
+#. Install and start rabbitmq via brew:
+
+   .. code-block:: shell
+
+      brew install rabbitmq && brew services start rabbitmq
+
+#. You will need to have `Docker installed <https://docs.docker.com/docker-for-mac/install/>`__ and running.
+#. Create Postgres db using the following command:
+   
+   .. code-block:: shell
+
+      docker run --name lookit-postgres -d -e POSTGRES_HOST_AUTH_METHOD="trust" -e POSTGRES_DB="lookit" -p 5432:5432 postgres:9.6
+
+#. To help with installing a specific version of python, we'll need to `install asdf <https://asdf-vm.com/#/core-manage-asdf?id=install>`__. 
+#. Add Python plugin using the following command.  Here's `documentation <https://github.com/danhper/asdf-python>`__ to help with errors:
+
+   .. code-block:: shell
+
+      asdf plugin-add python
+
+#. At the root of the project, install python.  The `asdf Python plugin docs <https://github.com/danhper/asdf-python>`__ can help install older versions of python:
+
+   .. code-block:: shell
+
+      asdf install
```

Comment:
does this automatically install the correct version of python (so that we no longer need to document that it should be 3.8)?

Reply:
The version of python that `asdf` installs is pulled from the `.tool-versions` file.